docs/source/en/tutorials/tutorial_overview.mdx

```diff
@@ -0,0 +1,23 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Overview
+
+Welcome to 🧨 Diffusers! If you're new to diffusion models and generative AI, and want to learn more, then you've come to the right place. These beginner-friendly tutorials are designed to provide a gentle introduction to diffusion models and help you understand the library fundamentals - the core components and how 🧨 Diffusers is meant to be used. 
+
+You'll learn how to use a pipeline for inference to rapidly generate samples, and then deconstruct that pipeline to really understand how to use the library as a modular toolbox for building your own diffusion systems. Then in the next lesson, you'll learn how to train your own diffusion model to generate what you want.
```

Comment:
are you referring to the "Understanding pipelines, models and schedulers" doc? are we going to move it to this section?

Reply:
Same question.

Also, should we briefly mention our dedicated training section that shows how to perform more advanced forms for training with diffusion models?

Reply:
> are you referring to the "Understanding pipelines, models and schedulers" doc? are we going to move it to this section?

Yeah, I think it'll be really nice to move that to the tutorials since it is such a super important doc for understanding how to use the library, and we want to give it maximum visibility.

> Also, should we briefly mention our dedicated training section that shows how to perform more advanced forms for training with diffusion models?

I think its ok if we don't mention that in the overview - since this is just the tutorial section - and we want the learner to focus on one thing first. But at the end of the training doc, we do have a [Next steps](https://huggingface.co/docs/diffusers/main/en/tutorials/basic_training#next-steps) section linking to the other forms of training available!